examples/datadog-express-apm/container-agent/src/sdk-v3.js

```diff
@@ -1,38 +1,38 @@
-const { ClientBuilder } = require('@commercetools/sdk-client-v2')
+const { ClientBuilder } = require('@commercetools/ts-client')
 const { createTelemetryMiddleware } = require('@commercetools/ts-sdk-apm')
 const { createApiBuilderFromCtpClient } = require('@commercetools/platform-sdk')
 const fetch = require('node-fetch')
 
 const projectKey = process.env.CTP_PROJECT_KEY
 const authMiddlewareOptions = {
-  host: 'https://auth.europe-west1.gcp.commercetools.com',
+  host: process.env.CTP_AUTH_URL,
   projectKey,
   credentials: {
     clientId: process.env.CTP_CLIENT_ID,
     clientSecret: process.env.CTP_CLIENT_SECRET,
-    // user: {
-    //   username: process.env.CTP_CLIENT_USERNAME,
-    //   password: process.env.CTP_CLIENT_PASSWORD,
-    // },
+    user: {
+      username: process.env.CTP_CLIENT_USERNAME,
+      password: process.env.CTP_CLIENT_PASSWORD,
+    },
```

Comment:
Since we are using the `ClientCredentialsFlow` you can remove this part of the code, it is not necessary